main.go

```diff
@@ -27,6 +27,11 @@ type GKE struct {
 	SecretTemplate string                 `json:"secret_template"`
 	Vars           map[string]interface{} `json:"vars"`
 	Secrets        map[string]interface{} `json:"secrets"`
+
+	// SecretsBase64 holds secret values which are already base64 encoded and
+	// thus don't need to be re-encoded as they would be if they were in
+	// the Secrets field.
+	SecretsBase64 map[string]interface{} `json:"secrets_base64"`
```

Comment:
Take a look at https://github.com/NYTimes/drone-gke/pull/19/commits/811fd325fa52c89a85f673e1c3581b85cb6e3286 to see the change from `interface{}` to `string`.